src/v3/src/components/ReminderPrompt/ReminderPrompt.tsx

```diff
@@ -42,23 +42,47 @@ const ReminderPrompt: UISchemaElementComponent<{
   const [show, setShow] = useState<boolean>(false);
   const timerRef = useRef<number | undefined>();
 
+  const RESEND_TIMESTAMP_SESSION_STORAGE_KEY = 'osw-oie-resend-timestamp';
+  const removeResendTimestamp = () => {
+    sessionStorage.removeItem(RESEND_TIMESTAMP_SESSION_STORAGE_KEY);
+  };
+  const setResendTimestamp = (token: string) => {
+    sessionStorage.setItem(RESEND_TIMESTAMP_SESSION_STORAGE_KEY, token);
+  };
+
+  // eslint-disable-next-line arrow-body-style
+  const getResendTimestamp = (): string | null => {
+    return sessionStorage.getItem(RESEND_TIMESTAMP_SESSION_STORAGE_KEY);
+  };
+
   const startTimer = () => {
+    setShow(false);
+    const timeStamp = getResendTimestamp();
+    if (!timeStamp) {
+      setResendTimestamp(Date.now().toString());
+    }
     if (timerRef) {
-      window.clearTimeout(timerRef.current);
+      window.clearInterval(timerRef.current);
```

Comment:
What happens when you omit the `window` part. 

Based on light research, `window.clearInterval` and `clearInterval` are equivalent so you should be able to leave off the `window` part